test/logging/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM python
+COPY log.py /
```

Comment:
Copying & running log.py from container / (root) ? why not from user home ?

Reply:
Didn't get this comment.

Reply:
>Copying & running log.py from container / (root) ?

Container also would run with a $USER ? If yes, shouldnt it use its $HOME to copy & execute ?